cluster/plan.go

```diff
@@ -580,6 +608,7 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 			"/dev:/host/dev:rprivate",
 			"/var/log/containers:/var/log/containers:z",
 			"/var/log/pods:/var/log/pods:z",
+			"/var/log/calico/cni:/var/log/calico/cni:z",
```

Comment:
Does adding `"/var/log/calico/cni:/var/log/calico/cni:z` here make it always to be in the list of "Binds" which invalidates the above version-checking logic?